src/ChcInterpreter.cc

```diff
@@ -7,6 +7,7 @@
 #include "ChcInterpreter.h"
 #include "Normalizer.h"
 #include "Validator.h"
+#include "engine/PDKind.h"
```

Comment:
Please move the include to the place where it belongs (engine group, alphabetical order).